[AbnormalSecurity] Expand integration commands

wolyslager opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

Created a new pull request as the[ previous on](https://github.com/demisto/content/pull/29558/files)e got muddled with merge conflict issues 

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

wolyslager commented:
hey @sapirshuker - I've been working with @MichaelYochpaz on this pr for a couple of weeks (I created a new one because the other one just got too messy with the merge conflicts and commit history) but its essentially the same changes. Just wanted to give you a heads up.

MichaelYochpaz reviewed (commented):
Hey, a few additional notes (most were on the previous PR from the previous review, and were resolved for some reason).

Please also note that the PR [fails a validation](https://app.circleci.com/pipelines/github/demisto/content/307058/workflows/f74e2eb2-599c-4691-a136-09e7c76fea5d/jobs/722440) for missing a version bump on the `pack_metadata.json` file, and that there are [some minor linting issues that need to be addressed](https://app.circleci.com/pipelines/github/demisto/content/307058/workflows/f74e2eb2-599c-4691-a136-09e7c76fea5d/jobs/722442).

wolyslager commented on Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py:
```diff
@@ -529,18 +634,215 @@ def submit_false_positive_report_command(client, args):
+    command_results = CommandResults(
+        readable_output=markdown,
+        outputs_prefix='AbnormalSecurity.VendorActivity',
+        outputs_key_field="",
```
There isn't for this one :(

MichaelYochpaz commented:
@wolyslager Looks good. Just fix the [failing validations](https://app.circleci.com/pipelines/github/demisto/content/307345/workflows/a36759c2-b9ad-44ee-a0b2-c0d4356319c1/jobs/722746) and [unit-tests](https://app.circleci.com/pipelines/github/demisto/content/307345/workflows/a36759c2-b9ad-44ee-a0b2-c0d4356319c1/jobs/722744).

Note that you've changed the release notes of an existing version ([2_0_27.md](https://github.com/demisto/content/pull/29994/files#diff-8d4c5a2a647f6a1f763235c75334af2487d0899290a24cac5dfa005f99f589dc)) instead of creating a new one (and the original one of a previous version should remain untouched).

I'd recommend just reverting the changes to just revert the changes you made to the release notes file & `pack_metadata.json` and run `demisto-sdk update-release-notes -i Packs/AbnormalSecurity -u minor`.
Or you can do that manually and just copy the changed to a new file named `2_1_0.md` and revert the changes on `2_0_27.md`.

Also, while making the changes, pull & merge from master (if you see any conflicts to unrelated files like last time don't merge) so that we can run the build after the changes and assure it passes.

wolyslager commented:
@MichaelYochpaz I've fixed the failing tests and created a new release notes file (2_1_0) and reverted the changes on the previous one (2_0_27). I had to manually update the pack metadata to match 2_1_0. I also pulled an merged from master and had no conflicts 😄  - let me know if I need to do anything else! 🙏

MichaelYochpaz commented:
@wolyslager There's a [validation error](https://app.circleci.com/pipelines/github/demisto/content/307570/workflows/c2fbb43c-11c4-4dad-901c-6549f0d71c3a/jobs/722905) - 

> [2023-10-04 14:23:07,126] [ERROR] Packs/AbnormalSecurity/ReleaseNotes/2_1_0.md: [RN111] - The 2_1_0.md release notes file contains incompatible Docker images:
> - Abnormal Security Event Collector: Release notes file has dockerimage: demisto/python3:3.10.13.74666 but the YML file has dockerimage: demisto/python3:3.10.13.75921
> - Abnormal Security: Release notes file has dockerimage: demisto/python3:3.10.13.74666 but the YML file has dockerimage: demisto/python3:3.10.13.75921
> To fix this please run: 'demisto-sdk update-release-notes -i Packs/AbnormalSecurity'

The Docker version specified in the release-notes doesn't match the one on the YAML file.
Other than that there don't seem to be any issues :)

MichaelYochpaz reviewed (approved):
Looks good, good job!
